randovania/data/json_data/prime3.txt

```diff
@@ -805,7 +833,12 @@ Asset id: 7569990525320728426
 > Dock to Repair Bay; Heals? False
   * Normal Door to Repair Bay/Dock to Stairwell
   > Dock to Weapons Cache
-      Plasma Beam and Energy Cells Total ≥ 2
+      After Stairwell Ramps
+  > Event - Stairwell Ramps
+      Plasma Beam and Energy Cells Total ≥ 3
+
+> Event - Stairwell Ramps; Heals? False
+  * Event Stairwell Ramps
 
 ----------------
 Weapons Cache
```

Comment:
Xenoresearch Lab (Below) is missing a connection

Reply:
I left that connection out intentionally since it's effectively a one way path. If we want to add a connection, I guess we'd want to add another event that opens up that tunnel, and make that event require lasso / 2 energy cells / missile?

Reply:
I stand corrected. A connection isn't necessary. The same could be said for Ventilation Shaft (below) as well then. In that room, connection to Xenoresearch should be Impossible, not Trivial, since it's effectively a one-way path.